### ISSUE TITLE
Ligature Request: "TODO:" "WARN:"

### ISSUE THREAD

**Issue by orta** [tonsky/FiraCode#339]
When combined words like TODO and WARN could move to a darker font weight. Example mockup:

![screen shot 2016-12-21 at 2 02 18 pm](https://cloud.githubusercontent.com/assets/49038/21402368/1f802280-c786-11e6-9cc8-1da809586a75.png)

**Comment by glebd** [comment]
I'd prefer the way PragmataPro treats [WARNING] or [ERROR]: 
<img width="566" alt="screen shot 2016-12-22 at 00 57 04" src="https://cloud.githubusercontent.com/assets/32538/21411602/add1a742-c7e1-11e6-85bb-d6c908ba55a8.png">

**Comment by chharvey** [comment]
FWIW, here's [a list of all TODO-like keywords supported in Atom](https://atom.io/packages/language-todo).

**Comment by ScottKillen** [comment]
This can already be achieved to some degree in some editors. (e.g. in VS Code: [Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments))

**Comment by chharvey** [comment]
Yes, this can already be achieved with styling such as `font-weight: bold` in an editor stylesheet.
